packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -1,24 +1,17 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import { Virtuoso } from 'react-virtuoso';
-import { useSearchParams } from 'react-router-dom';
-
-import 'pages/discussions/index.scss';
-
 import app from '../../../state';
+import { Footer } from '../../footer';
 import Sublayout from '../../sublayout';
 import { PageLoading } from '../loading';
 import { RecentThreadsHeader } from './recent_threads_header';
 import { ThreadPreview } from './thread_preview';
 
-type DiscussionsPageProps = {
-  topicName?: string;
-};
+import 'pages/discussions/index.scss';
 
-const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
+function DiscussionsPage({ topicName, stageName }) {
```

Comment:
Why changing this to function? We want to keep consistency and consistency is const for components, not function